addon_service/external_service/models.py

```diff
@@ -26,6 +27,13 @@ class ExternalService(AddonsServiceBaseModel):
         validators=[validate_service_type],
         verbose_name="Service type",
     )
+    icon_name = models.FilePathField(
```

Comment:
Does this means the developers will be the one uploading/committing icon files to the codebase?